myth

```diff
@@ -177,147 +170,126 @@ if args.solv:
+            exitWithError(args.outform, "IPC initialization failed. Please verify that your local Ethereum node is running, or use the -i flag to connect to INFURA. \n" + str(e))
 
 # Database search ops
 
 if args.search or args.init_db:
-
     contract_storage = get_persistent_storage(mythril_dir)
-
-    if (args.search):
-
+    if args.search:
         try:
             contract_storage.search(args.search, searchCallback)
         except SyntaxError:
             exitWithError(args.outform, "Syntax error in search expression.")
-
-    elif (args.init_db):
-            try:
-                contract_storage.initialize(eth, args.sync_all)
-            except FileNotFoundError as e:
-                exitWithError(args.outform, "Error syncing database over IPC: " + str(e))
-            except ConnectionError as e:
-                exitWithError(args.outform, "Could not connect to RPC server. Make sure that your node is running and that RPC parameters are set correctly.")               
+    elif args.init_db:
+        try:
+            contract_storage.initialize(eth, args.sync_all)
+        except FileNotFoundError as e:
+            exitWithError(args.outform, "Error syncing database over IPC: " + str(e))
+        except ConnectionError as e:
+            exitWithError(args.outform, "Could not connect to RPC server. Make sure that your node is running and that RPC parameters are set correctly.")
 
     sys.exit()
 
-
 # Load / compile input contracts
 
 contracts = []
+address = None
 
-if (args.code):
+if args.code:
     address = util.get_indexed_address(0)
-
     contracts.append(ETHContract(args.code, name="MAIN"))
 
 # Get bytecode from a contract address
 
-elif (args.address):
-
+elif args.address:
     address = args.address
-
     if not re.match(r'0x[a-fA-F0-9]{40}', args.address):
         exitWithError(args.outform, "Invalid contract address. Expected format is '0x...'.")
 
     try:
         code = eth.eth_getCode(args.address)
-
-        if (code == "0x"):
-            exitWithError(args.outform, "Received an empty response from eth_getCode. Check the contract address and verify that you are on the correct chain.") 
-
     except FileNotFoundError as e:
         exitWithError(args.outform, "IPC error: " + str(e))
     except ConnectionError as e:
         exitWithError(args.outform, "Could not connect to RPC server. Make sure that your node is running and that RPC parameters are set correctly.")
     except Exception as e:
         exitWithError(args.outform, "IPC / RPC error: " + str(e))
-
-    contracts.append(ETHContract(code, name=args.address))
+    else:
+        if code == "0x":
+            exitWithError(args.outform, "Received an empty response from eth_getCode. Check the contract address and verify that you are on the correct chain.")
+        else:
+            contracts.append(ETHContract(code, name=args.address))
 
 # Compile Solidity source file(s)
 
-elif (len(args.solidity_file)):
-
+elif args.solidity_file:
```

Comment:
To check if a string or list `True` or `False`, we probably can use it directly without `len()`:  <https://stackoverflow.com/questions/53513/how-do-i-check-if-a-list-is-empty>